src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separatedByLines = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '');
+  const stringObj = {};
+
+  for (const item of separatedByLines) {
+    const separetedItem = item.split(':');
+    const key = separetedItem[0].trim();
+    const value = separetedItem[1].trim();
+
+    stringObj[key] = value;
+  }
+
+  return stringObj;
```

Comment:
The function `convertToObject` uses a `for...of` loop to iterate over the array `separatedByLines`. According to the checklist, loops should be avoided in this task in favor of iteration methods. Consider using `reduce` to transform the array into an object.